types/node/tsconfig.json

```diff
@@ -5,6 +5,7 @@
     ],
     "compilerOptions": {
         "module": "commonjs",
+        "target": "es6",
         "lib": [
             "es6"
```

Comment:
Could we remove the forward declarations of `MapConstructor`, `Iterable`,.. around line 60 as we use lib and target es6 now? Or are they still needed for some other reason?

Reply:
It says "// compat for TypeScript 1.8", so yeah.
I also think it's silly to add a bunch of members to `interface Global` since most globals still won't be accessible from there anyway.